INDICADORES DE MERCADO LABORAL MEDIANTE LA METODOLOGIA DANE: EXPLORACION ANALISIS DESCRIPTIVO DE LOS RESULTADOS DE LA GEIH 2024 (DICIEMBRE) 

JHOAN SEBASTIAN ARIAS ALZATE DIURNO ELECTIVA 1
HAROLD BERMUDEZ 
ANDRES FELIPE PEÑALOSA FRANCO

Importar Librerias

In [16]:
import pandas as pd
import numpy as np
from pathlib import Path

Obtener directorio de trabajo actual (opcional)

In [17]:
current_working_directory = Path.cwd()
current_working_directory

PosixPath('/workspaces/ANALISIS-DE-DATOS-UQ/Notebooks')

LEER LOS MODULOS NECESARIOS PARA EL CALCULO DE LOS INDICADORES DE MERCADO LABORAL:
Caracteristicas generales, seguridad social y educación
Ocupados 
Fuerza de trabajo
No ocupados

In [18]:
#Caracteristicas generales, seguridad social en salud y educacion
#Links a los conjuntos de datos
url_cgsse = 'https://github.com/niconomist98/DataAnalyticsUQ/raw/refs/heads/main/Datos/GEIH2025/cgsse.CSV'
url_ocupados ='https://github.com/niconomist98/DataAnalyticsUQ/raw/refs/heads/main/Datos/GEIH2025/Ocupados.CSV'
url_ft = 'https://github.com/niconomist98/DataAnalyticsUQ/raw/refs/heads/main/Datos/GEIH2025/Fuerza%20de%20trabajo.CSV'
url_desocupados = 'https://raw.githubusercontent.com/niconomist98/DataAnalyticsUQ/refs/heads/main/Datos/GEIH2025/No%20ocupados.CSV'

# CGSSE
df_cgsse = pd.read_csv(url_cgsse, encoding='latin-1', sep=';', 
                       low_memory=False)

#Ocupados
df_ocupados=pd.read_csv(url_ocupados, encoding='latin-1',sep=';',
                        low_memory=False)

#Fuerza de trabajo
df_ft=pd.read_csv(url_ft,
                  encoding='latin-1',sep=';',low_memory=False)
#No ocupados
df_desocupados=pd.read_csv(url_desocupados,encoding='latin-1',
                           sep=';',low_memory=False)


In [19]:
df_cgsse

,PERIODO,MES,PER,DIRECTORIO,SECUENCIA_P,ORDEN,HOGAR,REGIS,AREA,CLASE,...,P6170,P3041,P3042,P3042S1,P3042S2,P3043,P3043S1,P3038,P3039,POB_MAY18
0,20250626,6,2025,8238974,1,1,1,10,NaN,2,...,2.0,NaN,5.0,2.0,NaN,2.0,NaN,1.0,2.0,1.0
1,20250626,6,2025,8238974,1,2,1,10,NaN,2,...,2.0,NaN,5.0,1.0,NaN,1.0,NaN,2.0,1.0,1.0
2,20250626,6,2025,8238975,1,1,1,10,NaN,2,...,2.0,NaN,3.0,3.0,NaN,NaN,NaN,2.0,1.0,1.0
3,20250626,6,2025,8238975,1,2,1,10,NaN,2,...,2.0,NaN,1.0,0.0,NaN,NaN,NaN,2.0,1.0,1.0
4,20250626,6,2025,8238975,1,3,1,10,NaN,2,...,2.0,NaN,1.0,0.0,NaN,NaN,NaN,1.0,2.0,1.0
...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...
67851,20250626,6,2025,8283060,1,1,1,10,88.0,1,...,2.0,NaN,5.0,2.0,NaN,2.0,NaN,1.0,2.0,1.0
67852,20250626,6,2025,8283060,1,2,1,10,88.0,1,...,2.0,NaN,5.0,2.0,NaN,2.0,NaN,2.0,1.0,1.0
67853,20250626,6,2025,8283062,1,1,1,10,94.0,1,...,2.0,NaN,3.0,3.0,NaN,NaN,NaN,2.0,1.0,1.0
67854,20250626,6,2025,8283062,1,2,1,10,94.0,1,...,2.0,NaN,3.0,2.0,NaN,NaN,NaN,1.0,2.0,1.0


In [20]:
df_cgsse.columns 
#FEX_C18 -> FACTOR DE EXPANSION 
#CUANTAS PERSONAS ESTIMA EL DANE CON CARACTERISTICAS SIMILARES 

Index(['PERIODO', 'MES', 'PER', 'DIRECTORIO', 'SECUENCIA_P', 'ORDEN', 'HOGAR',
       'REGIS', 'AREA', 'CLASE', 'FEX_C18', 'DPTO', 'PT', 'P6016', 'P3271',
       'P6040', 'P6030S1', 'P6030S3', 'P6050', 'P6083', 'P6083S1', 'P6081',
       'P6081S1', 'P2057', 'P2059', 'P2061', 'P6080', 'P6080S1', 'P6080S1A1',
       'P6070', 'P6071', 'P6071S1', 'P6090', 'P6100', 'P6110', 'P6120',
       'P1906S1', 'P1906S2', 'P1906S3', 'P1906S4', 'P1906S5', 'P1906S6',
       'P1906S7', 'P1906S8', 'P6160', 'P6170', 'P3041', 'P3042', 'P3042S1',
       'P3042S2', 'P3043', 'P3043S1', 'P3038', 'P3039', 'POB_MAY18'],
      dtype='str')

In [21]:
df_cgsse['FEX_C18'].sum()

np.float64(52045001.000009045)

In [22]:
df_ft['FEX_C18'].sum()

np.float64(40653416.00000409)

In [23]:
OCUPADOS=df_ocupados['FEX_C18'].sum()

In [24]:
DESOCUPADOS =df_desocupados['FEX_C18'].sum()

In [25]:
(df_desocupados['FEX_C18'].sum())/(df_ft['FEX_C18'].sum())

np.float64(0.4157206208636081)

In [26]:
#P3271: CUAL FUE SU SEXO AL NACER; M=1 F=2
PH = df_cgsse[df_cgsse['P3271'] ==1]['FEX_C18'].sum()
PM = df_cgsse[df_cgsse['P3271'] ==2]['FEX_C18'].sum()
print(f'hombres;   {PH}')
print(f'mujeres;   {PM}')


hombres;   25304270.000003804
mujeres;   26740731.000005245


In [27]:
FT = df_cgsse['FEX_C18']

In [28]:
PH/PT

NameError: name 'PT' is not defined

In [ ]:
PM/PT

NameError: name 'PT' is not defined

In [38]:
#CALCULAR POBLACION TOTAL DEL QUINDIO
Pquindio = df_cgsse[df_cgsse['DPTO'] ==63]['FEX_C18'].sum()
Pquindio

np.float64(603068.649439007)

In [39]:
PquindioH = df_cgsse[(df_cgsse['DPTO'] ==63) & (df_cgsse['P3271']==1)]['FEX_C18'].sum()
print(f'Hombres del Quindio:    {PquindioH}')

Hombres del Quindio:    297925.64044618


In [40]:
PquindioM = df_cgsse[(df_cgsse['DPTO'] ==63) & (df_cgsse['P3271']==2)]['FEX_C18'].sum()
print(f'Mujeres del Quindio:    {PquindioM}')

Mujeres del Quindio:    305143.00899282703


In [41]:
DESOCUPADOS/OCUPADOS

np.float64(0.7115099996821281)

In [42]:
df_ft.columns

Index(['PERIODO', 'MES', 'PER', 'DIRECTORIO', 'SECUENCIA_P', 'ORDEN', 'HOGAR',
       'REGIS', 'AREA', 'CLASE', 'FEX_C18', 'DPTO', 'FT', 'FFT', 'PET',
       'P6240', 'P6240S1', 'P6240S2', 'P6250', 'P6260', 'P6260S1', 'P6260S1A1',
       'P6260S2', 'P6270', 'P6280', 'P3362S1', 'P3362S2', 'P3362S3', 'P3362S4',
       'P3362S5', 'P3362S6', 'P3362S7', 'P3362S8', 'P3362S7A1', 'P6300',
       'P6310', 'P6310S1', 'P6320', 'P6330', 'P6340', 'P6350', 'P6351'],
      dtype='str')

TOTAL NACIONAL

In [43]:
#POBLACION TOTAL
PTN =df_cgsse['FEX_C18'].sum()
print(f'poblacion total nacional    {PTN}')

poblacion total nacional    52045001.000009045


In [44]:
#POBLACION EN EDAD DE TRABAJAR DESAGREGADA
df_cgsse[df_cgsse['P6040']>=15]['FEX_C18'].sum()


np.float64(40653416.00000409)

In [45]:
#POBLACION TOTAL NACIONAL EN EDAD DE TRABAJAR
PET= df_ft[df_ft['PET']==1]['FEX_C18'].sum()
print(f'Poblacion en edad de trabajar   {PET}')

Poblacion en edad de trabajar   40653416.00000409


In [46]:
#POBLACION QUE NO ESTA EN EDAD DE TRABAJAR
PENOT = df_cgsse[df_cgsse['P6040'] < 15 ]['FEX_C18'].sum()
print(f'poblacion en edad de no trabajar     {PENOT}')

poblacion en edad de no trabajar     11391585.000004958


In [47]:
PoTotal = PENOT + PET
print(f'suma de la poblacion total  {PoTotal}')

suma de la poblacion total  52045001.000009045


In [48]:
#POBLACION ECONOMICAMENTE ACTIVA
#Colocamos = 1 ya que este nos indica la fuerza de trabajo
#que solamente estara activa
PEA = df_ft[(df_ft['FT']==1)]['FEX_C18'].sum()
print(f'POBLACION ECONOMICAMENTE ACTIVA     {PEA}')


POBLACION ECONOMICAMENTE ACTIVA     25974846.22356785


In [49]:
#POBLACION ECONOMICAMENTE INACTIVA DESAGREGADA
PEINACDESAGRE= PET - PEA
print(f'Poblacion economicamente inactiva desagregada   {PEINACDESAGRE}')

Poblacion economicamente inactiva desagregada   14678569.77643624


In [50]:
#POBLACION ECONOMICAMENTE INACTIVA
PEI = df_ft[(df_ft['FFT']==1)]['FEX_C18'].sum()
print(f'POBLACION ECONOMICAMENTE INACTIVA {PEI}')

POBLACION ECONOMICAMENTE INACTIVA 14678569.776436238


In [ ]:
#
PEA + PEI

np.float64(40653416.00000409)

In [51]:
#CANTIDAD DE PERSONAS OCUPADAS
PO = df_ocupados[(df_ocupados['OCI'])==1]['FEX_C18'].sum()
print(f'POBLACION OCUPADA   {PO}')

POBLACION OCUPADA   23752952.660255846


In [52]:
#TASA DE OCUPACION = (POCUP/PET*100)
tasa_ocupacion = (PO)/(PET)*100
print(f'TASA DE OCUPACION   {tasa_ocupacion}')

TASA DE OCUPACION   58.427937913639184


In [53]:
PODESEMP = df_desocupados[df_desocupados['DSI']==1]['FEX_C18'].sum()
print(f'POBLACION NO OCUPADA    {PODESEMP}')

POBLACION NO OCUPADA    2221893.563312005


In [54]:
TASA_DESM = (PODESEMP/PEA)*100
print(f'Tasa de desempleo   {TASA_DESM}')

Tasa de desempleo   8.554020086155528


Calcule las tasas de desempleo para hombres y mujeres en Risaralda, Caldas , Tolima y Quindío. ¿Cuál es el departamento con mayor y menor tasa de desempleo de los 4 ? 

TASA DE DESEMPLEO PARA HOMBRES Y MUJERES EN RISARALDA

In [55]:
pob_desm_ris = df_desocupados[df_desocupados['DPTO']==66]['FEX_C18'].sum()
print(f'POBLACION NO OCUPADA RISARALDA    {pob_desm_ris}')

POBLACION NO OCUPADA RISARALDA    372420.82403131097


AL DataFrame de ocupados, le pegamos las llaves de busqueda para la discriminacion de la persona['DIRECTORIO','SECUENCIA_P','ORDEN'], y luego la categoria de busqueda [P3271] que me discrimina el genero en esa base de datos la cual no cuenta con genero, para poder discriminar entre hombres y mujeres desemplead@s en la ciudad de risaralda

In [56]:
df_o_genero_ocup = pd.merge(df_ocupados, df_cgsse[['DIRECTORIO','SECUENCIA_P','ORDEN','P3271']],
                       on=['DIRECTORIO','SECUENCIA_P','ORDEN'], how='left')

hacemos lo mismo con los desocupados para la correcta discriminacion de genero

In [57]:
df_o_genero_des = pd.merge(df_desocupados, df_cgsse[['DIRECTORIO','SECUENCIA_P','ORDEN','P3271']],
                       on=['DIRECTORIO','SECUENCIA_P','ORDEN'], how='left')

DATAFRAME FUERZA DE TRABAJO POR GENERO

In [58]:
df_o_genero_ft =  pd.merge(df_ft, df_cgsse[['DIRECTORIO','SECUENCIA_P','ORDEN','P3271']],
                       on=['DIRECTORIO','SECUENCIA_P','ORDEN'], how='left')

In [59]:
PDRIS = df_desocupados[(df_desocupados['DPTO']==66) & (df_desocupados['DSI']==1)]['FEX_C18'].sum()
print(f'TOTAL DE POBLACION DESEMPLEADA DE RISARALDA TRAIDA DIRECTAMENTE SIN DISCRIMINAR EL GENEREO  {PDRIS}')

TOTAL DE POBLACION DESEMPLEADA DE RISARALDA TRAIDA DIRECTAMENTE SIN DISCRIMINAR EL GENEREO  39677.403473861996


POBLACION DESEMPLEADA POR GENEROS EN RISARALDA

In [60]:
pob_desm_ris_h = df_o_genero_des[(df_o_genero_des['DPTO']==66) & (df_o_genero_des['DSI']==1) & (df_o_genero_des['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN RISARALDA       {pob_desm_ris_h}')

POBLACION DESMPLEADA HOMBRES EN RISARALDA       18000.60370715


In [61]:
pob_desm_ris_M = df_o_genero_des[(df_o_genero_des['DPTO']==66) & (df_o_genero_des['DSI']==1) & (df_o_genero_des['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN RISARALDA       {pob_desm_ris_M}')

POBLACION DESMPLEADA MUJERES EN RISARALDA       21676.799766712


In [62]:
PoTotalRisSum = pob_desm_ris_M + pob_desm_ris_h
print(f'SUMA DE LA POBLACION TOTAL DESEMPLEADA DE RISARALDA {PoTotalRisSum}')

SUMA DE LA POBLACION TOTAL DESEMPLEADA DE RISARALDA 39677.403473861996


POBLACION ECONOMICAMENTE ACTIVA PARA RISARALDA

In [63]:
PEA_ris = df_ft[(df_ft['DPTO']==66) & (df_ft['FT']==1)]['FEX_C18'].sum()
print(f'POBLACION ECONOMICAMENTE ACTIVA RISARALDA SIN DISCRIMINAR GENERO    {PEA_ris}')

POBLACION ECONOMICAMENTE ACTIVA RISARALDA SIN DISCRIMINAR GENERO    499500.23176734603


In [64]:
PEA_Hom = df_o_genero_ft[(df_o_genero_ft['DPTO']==66) & (df_o_genero_ft['FT']==1) & (df_o_genero_ft['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION ECONOMICAMENTE ACTIVA DE HOMBRES EN RISARALDA       {PEA_Hom}')

POBLACION ECONOMICAMENTE ACTIVA DE HOMBRES EN RISARALDA       296342.328368219


In [65]:
PEA_Muj = df_o_genero_ft[(df_o_genero_ft['DPTO']==66) & (df_o_genero_ft['FT']==1) & (df_o_genero_ft['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION ECONOMICAMENTE ACTIVA DE MUJERES EN RISARALDA       {PEA_Muj}')

POBLACION ECONOMICAMENTE ACTIVA DE MUJERES EN RISARALDA       203157.903399127


In [66]:
SUMADEPOEAR = PEA_Muj+PEA_Hom
print(f'SUMA DE LA PEA DE RISRALD   {SUMADEPOEAR}')

SUMA DE LA PEA DE RISRALD   499500.23176734603


In [67]:
#TASA DE DESEMPLEO HOMBRES RISARALDA
TASA_DES_RIS_H = (pob_desm_ris_h/PEA_Hom)*100
print(f'TASA DE DESEMPLEO PARA HOMBRES EN RISARALD  {TASA_DES_RIS_H}')

TASA DE DESEMPLEO PARA HOMBRES EN RISARALD  6.074260064793518


In [68]:
TASA_DES_RIS_M = (pob_desm_ris_M/PEA_Muj)*100
print(f'TASA DE DESEMPLEO PARA MUJERES EN RISARALDA {TASA_DES_RIS_M}')

TASA DE DESEMPLEO PARA MUJERES EN RISARALDA 10.669926891362644


In [69]:
DESEMPLEO_RISARALDA = (PDRIS/PEA_ris)*100
print(f'TASA DE DESEMPLEO RISARALS  {DESEMPLEO_RISARALDA}')

TASA DE DESEMPLEO RISARALS  7.943420433154609


TASA DE DESEMPLEO PARA HOMBRES Y MUJERES EN CALDAS

POBLACION DESEMPLEADA CALDAS

In [70]:
pob_desem_caldas = df_desocupados[(df_desocupados['DPTO']==17) & (df_desocupados['DSI']==1)]['FEX_C18'].sum()
print(f'POBLACION DESEMPLEADA CALDAS    {pob_desem_caldas}')

POBLACION DESEMPLEADA CALDAS    35440.65149664499


In [71]:
pob_desm_CALDAS_H = df_o_genero_des[(df_o_genero_des['DPTO']==17) & (df_o_genero_des['DSI']==1) & (df_o_genero_des['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN CALDAS       {pob_desm_CALDAS_H}')
pob_desm_CALDAS_M = df_o_genero_des[(df_o_genero_des['DPTO']==17) & (df_o_genero_des['DSI']==1) & (df_o_genero_des['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN CALDAS       {pob_desm_CALDAS_M}')
print(f'SUMA POBLACION DESEMPLEADA DE CALDDAS, TIENE QUE SER IGUAL AL DATO TRAIDO DIRECTAMENTE DEL DANE {pob_desm_CALDAS_H+pob_desm_CALDAS_M}')

POBLACION DESMPLEADA HOMBRES EN CALDAS       18226.160047418998
POBLACION DESMPLEADA MUJERES EN CALDAS       17214.491449226003
SUMA POBLACION DESEMPLEADA DE CALDDAS, TIENE QUE SER IGUAL AL DATO TRAIDO DIRECTAMENTE DEL DANE 35440.651496645005


POBLACION ECONOMICAMENTE ACTIVA PARA HOMBRES Y MUJERES EN CALDAS

In [72]:
PEA_CALDAS = df_ft[(df_ft['DPTO']==17) & (df_ft['FT']==1)]['FEX_C18'].sum()
print(f'POBLACION ECONOMICAMENTE ACTIVA DE CALDAS   {PEA_CALDAS}')

POBLACION ECONOMICAMENTE ACTIVA DE CALDAS   550654.001544983


In [73]:
PEA_Hom_CALDAS_H = df_o_genero_ft[(df_o_genero_ft['DPTO']==17) & (df_o_genero_ft['FT']==1) & (df_o_genero_ft['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION EA HOMBRES EN CALDAS       {PEA_Hom_CALDAS_H}')
PEA_Hom_CALDAS_M = df_o_genero_ft[(df_o_genero_ft['DPTO']==17) & (df_o_genero_ft['FT']==1) & (df_o_genero_ft['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION EA MUJERES EN CALDAS       {PEA_Hom_CALDAS_M}')
print(f'POBLACION EA EN CALDAS TOTAL, TIENE QUE SER IGUAL AL DATO TRAIDO DIRECTAMENTE      {PEA_Hom_CALDAS_M+PEA_Hom_CALDAS_H}')

POBLACION EA HOMBRES EN CALDAS       326447.467769983
POBLACION EA MUJERES EN CALDAS       224206.533775
POBLACION EA EN CALDAS TOTAL, TIENE QUE SER IGUAL AL DATO TRAIDO DIRECTAMENTE      550654.001544983


In [74]:
#TASA DESEMPLEO CALDAS
TDCH= (pob_desm_CALDAS_H/PEA_Hom_CALDAS_H)*100
TDCM= (pob_desm_CALDAS_M/PEA_Hom_CALDAS_M)*100
print(f'TASA DE DESEMPLEO HOMBRES EN CALDAS    {TDCH}')
print(f'TASA DE DESEMPLEO MUJERES EN CALDAS    {TDCM}')
TASA_DESEM_CALDAS = (pob_desem_caldas/PEA_CALDAS)*100
print(f'TASA DESEMPLEO CALDAS   {TASA_DESEM_CALDAS}')

TASA DE DESEMPLEO HOMBRES EN CALDAS    5.5831831601957065
TASA DE DESEMPLEO MUJERES EN CALDAS    7.677961547053494
TASA DESEMPLEO CALDAS   6.436101689483471


TASA DE DESEMPLEO POR GENERO EN EL TOLIMA

In [75]:
POTODETOLIMA = df_desocupados[(df_desocupados['DPTO']==73) & (df_desocupados['DSI']==1)]['FEX_C18'].sum()
print(f'POBLACION TOTAL DESEMPLEADAD DEL TOLIMA, DATO TRAIDO DIRECTAMENTE DE LA BASE DE DATOS DEL DANE    {POTODETOLIMA}')
TDTOLIMA_H = df_o_genero_des[(df_o_genero_des['DPTO']==73) & (df_o_genero_des['DSI']==1) & (df_o_genero_des['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN TOLIMA       {TDTOLIMA_H}')
TDTOLIMA_M = df_o_genero_des[(df_o_genero_des['DPTO']==73) & (df_o_genero_des['DSI']==1) & (df_o_genero_des['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN TOLIMA       {TDTOLIMA_M}')
print(f'POBLACION TOTAL DESEMPLEADA DEL TOLIMA, TIENE QUE SER IGUAL AL DATO TRAIDO DIRECTAMENTE DE BASE DE DATOS DEL DANE  {TDTOLIMA_H + TDTOLIMA_M}')
PEATOTALTOLIMA = df_ft[(df_ft['DPTO']==73) & (df_ft['FT']==1)]['FEX_C18'].sum()
print(f'POBLACION TOTAL ECONOMICAMENTE ACTIVA DEL TOLIMA     {PEATOTALTOLIMA}')
PEA_T_H = df_o_genero_ft[(df_o_genero_ft['DPTO']==73) & (df_o_genero_ft['FT']==1) & (df_o_genero_ft['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION EA HOMBRES EN TOLIMA      {PEA_T_H}')
PEA_T_M = df_o_genero_ft[(df_o_genero_ft['DPTO']==73) & (df_o_genero_ft['FT']==1) & (df_o_genero_ft['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION EA MUJERES EN TOLIMA      {PEA_T_M}')
print(f'POBLACION TOTAL ECONOMICAMENTE ACTIVA DE TOLIMA, TIENE QUE COINCIDIR CON EL DATO DIRECTO    {PEA_T_H+PEA_T_M}')
TDTOLIMA_H1= (TDTOLIMA_H/PEA_T_H)*100
TDTOLIMA_M1= (TDTOLIMA_M/PEA_T_M)*100
print(f'TASA DE DESEMPLEO TOLIMA HOMBRES    {TDTOLIMA_H1}')
print(f'TASA DE DESEMPLEO TOLIMA MUJERES     {TDTOLIMA_M1}')
TASA_DESEM_TOLIMA = (POTODETOLIMA/PEATOTALTOLIMA)*100
print(f'TASA DE DESEMPLEO TOLIMA    {TASA_DESEM_TOLIMA}')

POBLACION TOTAL DESEMPLEADAD DEL TOLIMA, DATO TRAIDO DIRECTAMENTE DE LA BASE DE DATOS DEL DANE    80356.13369624
POBLACION DESMPLEADA HOMBRES EN TOLIMA       28255.96613253
POBLACION DESMPLEADA MUJERES EN TOLIMA       52100.16756370999
POBLACION TOTAL DESEMPLEADA DEL TOLIMA, TIENE QUE SER IGUAL AL DATO TRAIDO DIRECTAMENTE DE BASE DE DATOS DEL DANE  80356.13369624
POBLACION TOTAL ECONOMICAMENTE ACTIVA DEL TOLIMA     662641.5050232499
POBLACION EA HOMBRES EN TOLIMA      391325.28931057
POBLACION EA MUJERES EN TOLIMA      271316.21571268
POBLACION TOTAL ECONOMICAMENTE ACTIVA DE TOLIMA, TIENE QUE COINCIDIR CON EL DATO DIRECTO    662641.50502325
TASA DE DESEMPLEO TOLIMA HOMBRES    7.220582697916321
TASA DE DESEMPLEO TOLIMA MUJERES     19.20274740190366
TASA DE DESEMPLEO TOLIMA    12.126637569045808


TASA DE DESEMPLEO POR GENERO PARA QUINDIO

In [76]:
POTOTALQUINDIO = df_desocupados[(df_desocupados['DPTO']==63) & (df_desocupados['DSI']==1)]['FEX_C18'].sum()
print(f'poblacion total desmpleada en QUINDIO    {POTOTALQUINDIO}')
POQUINDIO_H = df_o_genero_des[(df_o_genero_des['DPTO']==63) & (df_o_genero_des['DSI']==1) & (df_o_genero_des['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN QUINDIO       {POQUINDIO_H}')
POQUINDIO_M = df_o_genero_des[(df_o_genero_des['DPTO']==63) & (df_o_genero_des['DSI']==1) & (df_o_genero_des['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN QUINDIO       {POQUINDIO_M}')
print(f'POBLACION TOTAL DESEMPLEADA DEL QUINDIO SUMA, TIENE QUE SER IGUAL AL DATO TRAIDO DIRECTAMENTE  {POQUINDIO_H + POQUINDIO_M}')
PEATOTALQUINDIO = df_ft[(df_ft['DPTO']==63) & (df_ft['FT']==1)]['FEX_C18'].sum()
print(f'poblacion EA para el QUINDIO     {PEATOTALQUINDIO}')
PEA_Q_H = df_o_genero_ft[(df_o_genero_ft['DPTO']==63) & (df_o_genero_ft['FT']==1) & (df_o_genero_ft['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION EA HOMBRES EN QUINDIO H      {PEA_T_H}')
PEA_Q_M = df_o_genero_ft[(df_o_genero_ft['DPTO']==63) & (df_o_genero_ft['FT']==1) & (df_o_genero_ft['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION EA MUJERES EN QUINDIO M       {PEA_T_M}')
print(f'POBLACION EA HOMBRES EN QUINDIO TOTAL       {PEA_Q_H+PEA_Q_M}')
TDQUINDIO_H= (POQUINDIO_H/PEA_Q_H)*100
TDQUINDIO_M= (POQUINDIO_M/PEA_Q_M)*100
print(f'TASA DESEMPLEO HOMBRES    {TDQUINDIO_H}')
print(f'TASA DESEMPLEO MUJERES    {TDQUINDIO_M}')
TASA_DESEM_QUINDIO = (POTOTALQUINDIO/PEATOTALQUINDIO)*100
print(f'TASA DE DESEMPLEO QUINDIO   {TASA_DESEM_QUINDIO}')

poblacion total desmpleada en QUINDIO    28488.055906734
POBLACION DESMPLEADA HOMBRES EN QUINDIO       17869.520960767004
POBLACION DESMPLEADA MUJERES EN QUINDIO       10618.534945967001
POBLACION TOTAL DESEMPLEADA DEL QUINDIO SUMA, TIENE QUE SER IGUAL AL DATO TRAIDO DIRECTAMENTE  28488.055906734007
poblacion EA para el QUINDIO     291511.75121534604
POBLACION EA HOMBRES EN QUINDIO H      391325.28931057
POBLACION EA MUJERES EN QUINDIO M       271316.21571268
POBLACION EA HOMBRES EN QUINDIO TOTAL       291511.751215346
TASA DESEMPLEO HOMBRES    10.233031773040919
TASA DESEMPLEO MUJERES    9.084531863781265
TASA DE DESEMPLEO QUINDIO   9.772524019345367


# Tasa de Desempleo para H y M

## Risaralda
- Hombres: 6.07% 
- Mujeres: 10.67%
- tasa total: 7.94%

## Caldas
- Hombres: 5.58%
- Mujeres: 7.68%
- tasa total: 6.44%
## Tolima
- Hombres: 7.22%
- Mujeres: 19.20%
- tasa total: 12.12%

## Quindío
- Hombres: 10.23%
- Mujeres: 9.08%
- tasa total: 9.77%

VIENDOLO DESDE UNA PERSPECTIVA GENERAL EL DPTO CON MAYOR TASA DE DESEMPLEO ES EL TOLIMA CON UN 12.12%
 Y EL DE MENOR TASA DE DESEMPLEO ES CALDAS 6.44%

## MAYOR Y MENOR TASA DE DESEMPLEO DISCRIMINADO POR GENEROS

#### HOMBRES : Quindío (10.23%) > Tolima (7.22%) > Risaralda (6.07%) > Caldas (5.58%)

#### MUJERES : Tolima (19.20%) > Risaralda (10.67%) > Quindío (9.08%) > Caldas (7.68%) 

#### LA TASA DE DESEMPLEO MAS ALTA PARA HOMBRES DE ESTOS 4 DPTOS ES EL QUINDIO: 10.23% Y LA MAS BAJA ES CALDAS: 5.58%

#### LA TASA DE DESEMPLEO MAS ALTA PARA MUJERES DE ESTOS 4 DPTOS ES EL TOLIMA: 19.20% Y LA MAS BAJA ES CALDAS: 7.68%

#### Tomando en consideración únicamente los siguientes grupos de edad: 

#### 18 a 24
#### 24 a 30
#### 30 a 40
#### 40 a 50 
#### 60 en adelante 

#### Para Risaralda, Caldas , Tolima y Quindío, cual es el grupo de edad con mayor y menor tasa de desempleo (indicar en cada departamento el grupo de de edad  con mayor y menor tasa de desempleo) ? 


In [77]:
df_ocu_edad_g = pd.merge(df_ocupados, df_cgsse[['DIRECTORIO','SECUENCIA_P','ORDEN','P3271','P6040']], on=['DIRECTORIO','SECUENCIA_P','ORDEN'], how='left')
df_des_edad_g = pd.merge(df_desocupados, df_cgsse[['DIRECTORIO','SECUENCIA_P','ORDEN','P3271','P6040']], on=['DIRECTORIO','SECUENCIA_P','ORDEN'], how='left')
df_ft_edad_g =  pd.merge(df_ft, df_cgsse[['DIRECTORIO','SECUENCIA_P','ORDEN','P3271','P6040']], on=['DIRECTORIO','SECUENCIA_P','ORDEN'], how='left')

In [78]:
DESEM_RISA_18_24_H = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 24) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN RISARALDA  18-24     {DESEM_RISA_18_24_H}') 

DESEM_RISA_24_30_H = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 24) & (df_des_edad_g['P6040'] < 30) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN RISARALDA  24-30     {DESEM_RISA_24_30_H}') 

DESEM_RISA_30_40_H = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 30) & (df_des_edad_g['P6040'] < 40) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN RISARALDA  30-40     {DESEM_RISA_30_40_H}') 

DESEM_RISA_40_50_H = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 40) & (df_des_edad_g['P6040'] < 50) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN RISARALDA  40-50     {DESEM_RISA_40_50_H}') 

DESEM_RISA_60_H = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 60) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN RISARALDA  60>       {DESEM_RISA_60_H}') 

POBLACION DESMPLEADA HOMBRES EN RISARALDA  18-24     1265.39563095
POBLACION DESMPLEADA HOMBRES EN RISARALDA  24-30     3400.1134832200005
POBLACION DESMPLEADA HOMBRES EN RISARALDA  30-40     4359.68135668
POBLACION DESMPLEADA HOMBRES EN RISARALDA  40-50     3288.25774319
POBLACION DESMPLEADA HOMBRES EN RISARALDA  60>       1396.80706724


In [79]:
DESEM_RISA_18_24_M = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 24) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN RISARALDA  18-24     {DESEM_RISA_18_24_M}') 

DESEM_RISA_24_30_M = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 24) & (df_des_edad_g['P6040'] < 30) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN RISARALDA  24-30     {DESEM_RISA_24_30_M}') 

DESEM_RISA_30_40_M = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 30) & (df_des_edad_g['P6040'] < 40) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN RISARALDA  30-40     {DESEM_RISA_30_40_M}') 

DESEM_RISA_40_50_M = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 40) & (df_des_edad_g['P6040'] < 50) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN RISARALDA  40-50     {DESEM_RISA_40_50_M}') 

DESEM_RISA_60_M = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 60) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN RISARALDA  60>       {DESEM_RISA_60_M}') 

POBLACION DESMPLEADA MUJERES EN RISARALDA  18-24     4683.346698672001
POBLACION DESMPLEADA MUJERES EN RISARALDA  24-30     4608.179859849999
POBLACION DESMPLEADA MUJERES EN RISARALDA  30-40     3203.3109072199995
POBLACION DESMPLEADA MUJERES EN RISARALDA  40-50     2763.4498996699995
POBLACION DESMPLEADA MUJERES EN RISARALDA  60>       824.6385680400001


In [80]:
PEA_RIS_18_24_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 24) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES RISARALDA 18-24      {PEA_RIS_18_24_H}')

PEA_RIS_24_30_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 24) & (df_ft_edad_g['P6040'] < 30) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES RISARALDA 24-30      {PEA_RIS_24_30_H}')

PEA_RIS_30_40_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 30) & (df_ft_edad_g['P6040'] < 40) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES RISARALDA 18-24      {PEA_RIS_30_40_H}')

PEA_RIS_40_50_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 40) & (df_ft_edad_g['P6040'] < 50) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES RISARALDA 18-24      {PEA_RIS_40_50_H}')

PEA_RIS_60_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 60) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES RISARALDA 18-24      {PEA_RIS_60_H}')

PEA HOMBRES RISARALDA 18-24      23561.375587050003
PEA HOMBRES RISARALDA 24-30      38197.1224518
PEA HOMBRES RISARALDA 18-24      72090.50188416001
PEA HOMBRES RISARALDA 18-24      50756.866190689994
PEA HOMBRES RISARALDA 18-24      40941.200880462


In [81]:
PEA_RIS_18_24_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 24) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES RISARALDA 18-24      {PEA_RIS_18_24_M}')

PEA_RIS_24_30_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 24) & (df_ft_edad_g['P6040'] < 30) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES RISARALDA 24-30      {PEA_RIS_24_30_M}')

PEA_RIS_30_40_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 30) & (df_ft_edad_g['P6040'] < 40) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES RISARALDA 30-40      {PEA_RIS_30_40_M}')

PEA_RIS_40_50_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 40) & (df_ft_edad_g['P6040'] < 50) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES RISARALDA 40-60      {PEA_RIS_40_50_M}')

PEA_RIS_60_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 60) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES RISARALDA 60         {PEA_RIS_60_M}')

PEA MUJERES RISARALDA 18-24      20888.226128447004
PEA MUJERES RISARALDA 24-30      25333.028554811
PEA MUJERES RISARALDA 30-40      56966.068297637
PEA MUJERES RISARALDA 40-60      35736.23224248799
PEA MUJERES RISARALDA 60         14075.27338353


In [82]:
#TASA DE DESEMPLEO RISARALDA
TASA_D_R_18_24_H = (DESEM_RISA_18_24_H/PEA_RIS_18_24_H)*100
TASA_D_R_24_30_H = (DESEM_RISA_24_30_H/PEA_RIS_24_30_H)*100
TASA_D_R_30_40_H = (DESEM_RISA_30_40_H/PEA_RIS_30_40_H)*100
TASA_D_R_40_50_H = (DESEM_RISA_40_50_H/PEA_RIS_40_50_H)*100
TASA_D_R_60_H = (DESEM_RISA_60_H/PEA_RIS_60_H)*100
TASA_D_R_18_24_M = (DESEM_RISA_18_24_M/PEA_RIS_18_24_M)*100
TASA_D_R_24_30_M = (DESEM_RISA_24_30_M/PEA_RIS_24_30_M)*100
TASA_D_R_30_40_M = (DESEM_RISA_30_40_M/PEA_RIS_30_40_M)*100
TASA_D_R_40_50_M = (DESEM_RISA_40_50_M/PEA_RIS_40_50_M)*100
TASA_D_R_60_M = (DESEM_RISA_60_M/PEA_RIS_60_M)*100
print(f'TASA DESEMPLEO HOMBRES RISARALDA 18-24    {TASA_D_R_18_24_H}')
print(f'TASA DESEMPLEO HOMBRES RISARALDA 24-30    {TASA_D_R_24_30_H}')
print(f'TASA DESEMPLEO HOMBRES RISARALDA 30-40    {TASA_D_R_30_40_H}')
print(f'TASA DESEMPLEO HOMBRES RISARALDA 40-50    {TASA_D_R_40_50_H}')
print(f'TASA DESEMPLEO HOMBRES RISARALDA 60    {TASA_D_R_60_H}')
print(f'TASA DESEMPLEO MUJERES RISARALDA 18-24    {TASA_D_R_18_24_M}')
print(f'TASA DESEMPLEO MUJERES RISARALDA 24-30    {TASA_D_R_24_30_M}')
print(f'TASA DESEMPLEO MUJERES RISARALDA 30-40    {TASA_D_R_30_40_M}')
print(f'TASA DESEMPLEO MUJERES RISARALDA 40-50    {TASA_D_R_40_50_M}')
print(f'TASA DESEMPLEO MUJERES RISARALDA 60    {TASA_D_R_60_M}')

TASA DESEMPLEO HOMBRES RISARALDA 18-24    5.370635624710712
TASA DESEMPLEO HOMBRES RISARALDA 24-30    8.901491172562851
TASA DESEMPLEO HOMBRES RISARALDA 30-40    6.047511451210919
TASA DESEMPLEO HOMBRES RISARALDA 40-50    6.47844910447435
TASA DESEMPLEO HOMBRES RISARALDA 60    3.411739365726777
TASA DESEMPLEO MUJERES RISARALDA 18-24    22.42098811968481
TASA DESEMPLEO MUJERES RISARALDA 24-30    18.190402501144533
TASA DESEMPLEO MUJERES RISARALDA 30-40    5.623191143336946
TASA DESEMPLEO MUJERES RISARALDA 40-50    7.7329078256449275
TASA DESEMPLEO MUJERES RISARALDA 60    5.858774785894676


In [83]:
#DESEMPLEADOS HOMBRES CALDAS
DESEM_CALDAS_18_24_H = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 24) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN CALDAS  18-24     {DESEM_CALDAS_18_24_H}') 

DESEM_CALDAS_24_30_H = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 24) & (df_des_edad_g['P6040'] < 30) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN CALDAS  24-30     {DESEM_CALDAS_24_30_H}') 

DESEM_CALDAS_30_40_H = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 30) & (df_des_edad_g['P6040'] < 40) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN CALDAS  30-40     {DESEM_CALDAS_30_40_H}') 

DESEM_CALDAS_40_50_H = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 40) & (df_des_edad_g['P6040'] < 50) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN CALDAS  40-50     {DESEM_CALDAS_40_50_H}') 

DESEM_CALDAS_60_H = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 60) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN CALDAS  60>       {DESEM_CALDAS_60_H}') 

#DESEMPLEADOS MUJERES CALDAS
DESEM_CALDAS_18_24_M = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 24) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN CALDAS  18-24     {DESEM_CALDAS_18_24_M}') 

DESEM_CALDAS_24_30_M = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 24) & (df_des_edad_g['P6040'] < 30) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN CALDAS  24-30     {DESEM_CALDAS_24_30_M}') 

DESEM_CALDAS_30_40_M = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 30) & (df_des_edad_g['P6040'] < 40) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN CALDAS  30-40     {DESEM_CALDAS_30_40_M}') 

DESEM_CALDAS_40_50_M = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 40) & (df_des_edad_g['P6040'] < 50) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN CALDAS  40-50     {DESEM_CALDAS_40_50_M}') 

DESEM_CALDAS_60_M = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 60) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN CALDAS  60>       {DESEM_CALDAS_60_M}')

#PEA HOMBRES CALDAS
PEA_CALDAS_18_24_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 24) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES CALDAS 18-24      {PEA_CALDAS_18_24_H}')

PEA_CALDAS_24_30_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 24) & (df_ft_edad_g['P6040'] < 30) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES CALDAS 24-30      {PEA_CALDAS_24_30_H}')

PEA_CALDAS_30_40_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 30) & (df_ft_edad_g['P6040'] < 40) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES CALDAS 18-24      {PEA_CALDAS_30_40_H}')

PEA_CALDAS_40_50_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 40) & (df_ft_edad_g['P6040'] < 50) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES CALDAS 18-24      {PEA_CALDAS_40_50_H}')

PEA_CALDAS_60_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 60) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES CALDAS 18-24      {PEA_CALDAS_60_H}')

#PEA MUJERES CALDAS
PEA_CALDAS_18_24_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 24) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 18-24      {PEA_CALDAS_18_24_M}')

PEA_CALDAS_24_30_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 24) & (df_ft_edad_g['P6040'] < 30) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 24-30      {PEA_CALDAS_24_30_M}')

PEA_CALDAS_30_40_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 30) & (df_ft_edad_g['P6040'] < 40) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 30-40      {PEA_CALDAS_30_40_M}')

PEA_CALDAS_40_50_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 40) & (df_ft_edad_g['P6040'] < 50) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 40-60      {PEA_CALDAS_40_50_M}')
PEA_CALDAS_60_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 60) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 60         {PEA_CALDAS_60_M}')

#TASA DE DESEMPLEO DE HOMBRES Y MUJERES
TASA_D_C_18_24_H = (DESEM_CALDAS_18_24_H/PEA_CALDAS_18_24_H)*100
TASA_D_C_24_30_H = (DESEM_CALDAS_24_30_H/PEA_CALDAS_24_30_H)*100
TASA_D_C_30_40_H = (DESEM_CALDAS_30_40_H/PEA_CALDAS_30_40_H)*100
TASA_D_C_40_50_H = (DESEM_CALDAS_40_50_H/PEA_CALDAS_40_50_H)*100
TASA_D_C_60_H = (DESEM_CALDAS_60_H/PEA_CALDAS_60_H)*100
TASA_D_C_18_24_M = (DESEM_CALDAS_18_24_M/PEA_CALDAS_18_24_M)*100
TASA_D_C_24_30_M = (DESEM_CALDAS_24_30_M/PEA_CALDAS_24_30_M)*100
TASA_D_C_30_40_M = (DESEM_CALDAS_30_40_M/PEA_CALDAS_30_40_M)*100
TASA_D_C_40_50_M = (DESEM_CALDAS_40_50_M/PEA_CALDAS_40_50_M)*100
TASA_D_C_60_M = (DESEM_CALDAS_60_M/PEA_CALDAS_60_M)*100
print(f'TASA DESEMPLEO HOMBRES CALDAS 18-24    {TASA_D_C_18_24_H}')
print(f'TASA DESEMPLEO HOMBRES CALDAS 24-30    {TASA_D_C_24_30_H}')
print(f'TASA DESEMPLEO HOMBRES CALDAS 30-40    {TASA_D_C_30_40_H}')
print(f'TASA DESEMPLEO HOMBRES CALDAS 40-50    {TASA_D_C_40_50_H}')
print(f'TASA DESEMPLEO HOMBRES CALDAS 60       {TASA_D_C_60_H}')
print(f'TASA DESEMPLEO MUJERES CALDAS 18-24    {TASA_D_C_18_24_M}')
print(f'TASA DESEMPLEO MUJERES CALDAS 24-30    {TASA_D_C_24_30_M}')
print(f'TASA DESEMPLEO MUJERES CALDAS 30-40    {TASA_D_C_30_40_M}')
print(f'TASA DESEMPLEO MUJERES CALDAS 40-50    {TASA_D_C_40_50_M}')
print(f'TASA DESEMPLEO MUJERES CALDAS 60       {TASA_D_C_60_M}')

POBLACION DESMPLEADA HOMBRES EN CALDAS  18-24     2690.1350818
POBLACION DESMPLEADA HOMBRES EN CALDAS  24-30     1480.30071959
POBLACION DESMPLEADA HOMBRES EN CALDAS  30-40     2390.6174671299996
POBLACION DESMPLEADA HOMBRES EN CALDAS  40-50     1936.1109277500002
POBLACION DESMPLEADA HOMBRES EN CALDAS  60>       1821.085924862
POBLACION DESMPLEADA MUJERES EN CALDAS  18-24     1858.618688844
POBLACION DESMPLEADA MUJERES EN CALDAS  24-30     2918.2968940680003
POBLACION DESMPLEADA MUJERES EN CALDAS  30-40     2644.0294699640003
POBLACION DESMPLEADA MUJERES EN CALDAS  40-50     2671.2552592300003
POBLACION DESMPLEADA MUJERES EN CALDAS  60>       722.6044061900001
PEA HOMBRES CALDAS 18-24      31639.114960834995
PEA HOMBRES CALDAS 24-30      45072.174665209
PEA HOMBRES CALDAS 18-24      60206.776366661
PEA HOMBRES CALDAS 18-24      56135.623151823005
PEA HOMBRES CALDAS 18-24      39350.830191491
PEA MUJERES CALDAS 18-24      17335.256766072
PEA MUJERES CALDAS 24-30      26887.961173701002

In [84]:
#DESEMPLEADOS HOMBRES TOLIMA
DESEM_TOLIMA_18_24_H = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 24) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN TOLIMA  18-24     {DESEM_TOLIMA_18_24_H}') 

DESEM_TOLIMA_24_30_H = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 24) & (df_des_edad_g['P6040'] < 30) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN TOLIMA  24-30     {DESEM_TOLIMA_24_30_H}') 

DESEM_TOLIMA_30_40_H = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 30) & (df_des_edad_g['P6040'] < 40) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN TOLIMA  30-40     {DESEM_TOLIMA_30_40_H}') 

DESEM_TOLIMA_40_50_H = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 40) & (df_des_edad_g['P6040'] < 50) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN TOLIMA  40-50     {DESEM_TOLIMA_40_50_H}') 

DESEM_TOLIMA_60_H = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 60) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN CALTOLIMAAS  60>       {DESEM_TOLIMA_60_H}') 

#DESEMPLEADOS MUJERES TOLIMA
DESEM_TOLIMA_18_24_M = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 24) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN TOLIMA  18-24     {DESEM_TOLIMA_18_24_M}') 

DESEM_TOLIMA_24_30_M = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 24) & (df_des_edad_g['P6040'] < 30) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN TOLIMA  24-30     {DESEM_TOLIMA_24_30_M}') 

DESEM_TOLIMA_30_40_M = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 30) & (df_des_edad_g['P6040'] < 40) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN TOLIMA  30-40     {DESEM_TOLIMA_30_40_M}') 

DESEM_TOLIMA_40_50_M = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 40) & (df_des_edad_g['P6040'] < 50) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN TOLIMA  40-50     {DESEM_TOLIMA_40_50_M}') 

DESEM_TOLIMA_60_M = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 60) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN TOLIMA  60>       {DESEM_TOLIMA_60_M}')

#PEA HOMBRES TOLIMA
PEA_TOLIMA_18_24_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 24) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES CALDAS 18-24      {PEA_TOLIMA_18_24_H}')

PEA_TOLIMA_24_30_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 24) & (df_ft_edad_g['P6040'] < 30) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES CALDAS 24-30      {PEA_TOLIMA_24_30_H}')

PEA_TOLIMA_30_40_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 30) & (df_ft_edad_g['P6040'] < 40) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES CALDAS 18-24      {PEA_TOLIMA_30_40_H}')

PEA_TOLIMA_40_50_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 40) & (df_ft_edad_g['P6040'] < 50) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES CALDAS 18-24      {PEA_TOLIMA_40_50_H}')

PEA_TOLIMA_60_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 60) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES CALDAS 18-24      {PEA_TOLIMA_60_H}')

#PEA MUJERES TOLIMA
PEA_TOLIMA_18_24_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 24) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 18-24      {PEA_TOLIMA_18_24_M}')

PEA_TOLIMA_24_30_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 24) & (df_ft_edad_g['P6040'] < 30) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 24-30      {PEA_TOLIMA_24_30_M}')

PEA_TOLIMA_30_40_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 30) & (df_ft_edad_g['P6040'] < 40) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 30-40      {PEA_TOLIMA_30_40_M}')

PEA_TOLIMA_40_50_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 40) & (df_ft_edad_g['P6040'] < 50) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 40-60      {PEA_TOLIMA_40_50_M}')

PEA_TOLIMA_60_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 60) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 60         {PEA_TOLIMA_60_M}')

#TASA DE DESEMPLEO DE HOMBRES Y MUJERES TOLIMA
TASA_D_T_18_24_H = (DESEM_TOLIMA_18_24_H/PEA_TOLIMA_18_24_H)*100
TASA_D_T_24_30_H = (DESEM_TOLIMA_24_30_H/PEA_TOLIMA_24_30_H)*100
TASA_D_T_30_40_H = (DESEM_TOLIMA_30_40_H/PEA_TOLIMA_30_40_H)*100
TASA_D_T_40_50_H = (DESEM_TOLIMA_40_50_H/PEA_TOLIMA_40_50_H)*100
TASA_D_T_60_H = (DESEM_TOLIMA_60_H/PEA_TOLIMA_60_H)*100
TASA_D_T_18_24_M = (DESEM_TOLIMA_18_24_M/PEA_TOLIMA_18_24_M)*100
TASA_D_T_24_30_M = (DESEM_TOLIMA_24_30_M/PEA_TOLIMA_24_30_M)*100
TASA_D_T_30_40_M = (DESEM_TOLIMA_30_40_M/PEA_TOLIMA_30_40_M)*100
TASA_D_T_40_50_M = (DESEM_TOLIMA_40_50_M/PEA_TOLIMA_40_50_M)*100
TASA_D_T_60_M = (DESEM_TOLIMA_60_M/PEA_TOLIMA_60_M)*100
print(f'TASA DESEMPLEO HOMBRES TOLIMA 18-24    {TASA_D_T_18_24_H}')
print(f'TASA DESEMPLEO HOMBRES TOLIMA 24-30    {TASA_D_T_24_30_H}')
print(f'TASA DESEMPLEO HOMBRES TOLIMA 30-40    {TASA_D_T_30_40_H}')
print(f'TASA DESEMPLEO HOMBRES TOLIMA 40-50    {TASA_D_T_40_50_H}')
print(f'TASA DESEMPLEO HOMBRES TOLIMA 60       {TASA_D_T_60_H}')
print(f'TASA DESEMPLEO MUJERES TOLIMA 18-24    {TASA_D_T_18_24_M}')
print(f'TASA DESEMPLEO MUJERES TOLIMA 24-30    {TASA_D_T_24_30_M}')
print(f'TASA DESEMPLEO MUJERES TOLIMA 30-40    {TASA_D_T_30_40_M}')
print(f'TASA DESEMPLEO MUJERES TOLIMA 40-50    {TASA_D_T_40_50_M}')
print(f'TASA DESEMPLEO MUJERES TOLIMA 60       {TASA_D_T_60_M}')

POBLACION DESMPLEADA HOMBRES EN TOLIMA  18-24     4900.54637214
POBLACION DESMPLEADA HOMBRES EN TOLIMA  24-30     5658.40330805
POBLACION DESMPLEADA HOMBRES EN TOLIMA  30-40     4860.57519364
POBLACION DESMPLEADA HOMBRES EN TOLIMA  40-50     3074.38095006
POBLACION DESMPLEADA HOMBRES EN CALTOLIMAAS  60>       3550.1444810800003
POBLACION DESMPLEADA MUJERES EN TOLIMA  18-24     4964.2725597300005
POBLACION DESMPLEADA MUJERES EN TOLIMA  24-30     4190.7857523
POBLACION DESMPLEADA MUJERES EN TOLIMA  30-40     13053.276290119999
POBLACION DESMPLEADA MUJERES EN TOLIMA  40-50     8140.342793829999
POBLACION DESMPLEADA MUJERES EN TOLIMA  60>       3618.24587873
PEA HOMBRES CALDAS 18-24      40398.77507737
PEA HOMBRES CALDAS 24-30      38606.391492400006
PEA HOMBRES CALDAS 18-24      73394.58552200999
PEA HOMBRES CALDAS 18-24      64755.00538769
PEA HOMBRES CALDAS 18-24      67426.06457866
PEA MUJERES CALDAS 18-24      21472.5610457
PEA MUJERES CALDAS 24-30      29336.82724937
PEA MUJERES CALD

In [85]:
#DESEMPLEADOS HOMBRES QUINDIO
DESEM_QUINDIO_18_24_H = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 24) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN QUINDIO  18-24     {DESEM_QUINDIO_18_24_H}') 

DESEM_QUINDIO_24_30_H = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 24) & (df_des_edad_g['P6040'] < 30) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN QUINDIO  24-30     {DESEM_QUINDIO_24_30_H}') 

DESEM_QUINDIO_30_40_H = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 30) & (df_des_edad_g['P6040'] < 40) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN QUINDIO  30-40     {DESEM_QUINDIO_30_40_H}') 

DESEM_QUINDIO_40_50_H = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 40) & (df_des_edad_g['P6040'] < 50) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN QUINDIO  40-50     {DESEM_QUINDIO_40_50_H}') 

DESEM_QUINDIO_60_H = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 60) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA HOMBRES EN QUINDIO  60>       {DESEM_QUINDIO_60_H}') 

#DESEMPLEADOS MUJERES QUINDIO
DESEM_QUINDIO_18_24_M = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 24) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN QUINDIO  18-24     {DESEM_QUINDIO_18_24_M}') 

DESEM_QUINDIO_24_30_M = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 24) & (df_des_edad_g['P6040'] < 30) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN QUINDIO  24-30     {DESEM_QUINDIO_24_30_M}') 

DESEM_QUINDIO_30_40_M = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 30) & (df_des_edad_g['P6040'] < 40) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN QUINDIO  30-40     {DESEM_QUINDIO_30_40_M}') 

DESEM_QUINDIO_40_50_M = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 40) & (df_des_edad_g['P6040'] < 50) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN QUINDIO  40-50     {DESEM_QUINDIO_40_50_M}') 

DESEM_QUINDIO_60_M = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 60) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN QUINDIO  60>       {DESEM_QUINDIO_60_M}')

#PEA HOMBRES QUINDIO
PEA_QUINDIO_18_24_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 24) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES QUINDIO 18-24      {PEA_QUINDIO_18_24_H}')

PEA_QUINDIO_24_30_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 24) & (df_ft_edad_g['P6040'] < 30) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES QUINDIO 24-30      {PEA_QUINDIO_24_30_H}')

PEA_QUINDIO_30_40_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 30) & (df_ft_edad_g['P6040'] < 40) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES QUINDIO 30-40      {PEA_QUINDIO_30_40_H}')

PEA_QUINDIO_40_50_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 40) & (df_ft_edad_g['P6040'] < 50) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES QUINDIO 40-50      {PEA_QUINDIO_40_50_H}')

PEA_QUINDIO_60_H = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 60) & (df_ft_edad_g['P3271']==1)]['FEX_C18'].sum()
print(f'PEA HOMBRES QUINDIO 60         {PEA_QUINDIO_60_H}')

#PEA MUJERES QUINDIO
PEA_QUINDIO_18_24_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 24) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES QUINDIO 18-24      {PEA_QUINDIO_18_24_M}')

PEA_QUINDIO_24_30_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 24) & (df_ft_edad_g['P6040'] < 30) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES QUINDIO 24-30      {PEA_QUINDIO_24_30_M}')

PEA_QUINDIO_30_40_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 30) & (df_ft_edad_g['P6040'] < 40) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES QUINDIO 30-40      {PEA_QUINDIO_30_40_M}')

PEA_QUINDIO_40_50_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 40) & (df_ft_edad_g['P6040'] < 50) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES QUINDIO 40-50      {PEA_QUINDIO_40_50_M}')
PEA_QUINDIO_60_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 60) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES QUINDIO 60         {PEA_QUINDIO_60_M}')

#TASA DE DESEMPLEO DE HOMBRES Y MUJERES
TASA_D_Q_18_24_H = (DESEM_QUINDIO_18_24_H/PEA_QUINDIO_18_24_H)*100
TASA_D_Q_24_30_H = (DESEM_QUINDIO_24_30_H/PEA_QUINDIO_24_30_H)*100
TASA_D_Q_30_40_H = (DESEM_QUINDIO_30_40_H/PEA_QUINDIO_30_40_H)*100
TASA_D_Q_40_50_H = (DESEM_QUINDIO_40_50_H/PEA_QUINDIO_40_50_H)*100
TASA_D_Q_60_H = (DESEM_QUINDIO_60_H/PEA_QUINDIO_60_H)*100
TASA_D_Q_18_24_M = (DESEM_QUINDIO_18_24_M/PEA_QUINDIO_18_24_M)*100
TASA_D_Q_24_30_M = (DESEM_QUINDIO_24_30_M/PEA_QUINDIO_24_30_M)*100
TASA_D_Q_30_40_M = (DESEM_QUINDIO_30_40_M/PEA_QUINDIO_30_40_M)*100
TASA_D_Q_40_50_M = (DESEM_QUINDIO_40_50_M/PEA_QUINDIO_40_50_M)*100
TASA_D_Q_60_M = (DESEM_QUINDIO_60_M/PEA_QUINDIO_60_M)*100
print(f'TASA DESEMPLEO HOMBRES QUINDIO 18-24    {TASA_D_Q_18_24_H}')
print(f'TASA DESEMPLEO HOMBRES QUINDIO 24-30    {TASA_D_Q_24_30_H}')
print(f'TASA DESEMPLEO HOMBRES QUINDIO 30-40    {TASA_D_Q_30_40_H}')
print(f'TASA DESEMPLEO HOMBRES QUINDIO 40-50    {TASA_D_Q_40_50_H}')
print(f'TASA DESEMPLEO HOMBRES QUINDIO 60       {TASA_D_Q_60_H}')
print(f'TASA DESEMPLEO MUJERES QUINDIO 18-24    {TASA_D_Q_18_24_M}')
print(f'TASA DESEMPLEO MUJERES QUINDIO 24-30    {TASA_D_Q_24_30_M}')
print(f'TASA DESEMPLEO MUJERES QUINDIO 30-40    {TASA_D_Q_30_40_M}')
print(f'TASA DESEMPLEO MUJERES QUINDIO 40-50    {TASA_D_Q_40_50_M}')
print(f'TASA DESEMPLEO MUJERES QUINDIO 60       {TASA_D_Q_60_M}')

POBLACION DESMPLEADA HOMBRES EN QUINDIO  18-24     2053.19408296
POBLACION DESMPLEADA HOMBRES EN QUINDIO  24-30     1672.0818796100002
POBLACION DESMPLEADA HOMBRES EN QUINDIO  30-40     4647.58065335
POBLACION DESMPLEADA HOMBRES EN QUINDIO  40-50     1672.4688434539999
POBLACION DESMPLEADA HOMBRES EN QUINDIO  60>       3765.644844189
POBLACION DESMPLEADA MUJERES EN QUINDIO  18-24     310.22422941
POBLACION DESMPLEADA MUJERES EN QUINDIO  24-30     3299.872696118
POBLACION DESMPLEADA MUJERES EN QUINDIO  30-40     1215.255157759
POBLACION DESMPLEADA MUJERES EN QUINDIO  40-50     2126.23707103
POBLACION DESMPLEADA MUJERES EN QUINDIO  60>       290.53275914
PEA HOMBRES QUINDIO 18-24      12619.526296720001
PEA HOMBRES QUINDIO 24-30      19496.8574117
PEA HOMBRES QUINDIO 30-40      37510.365950645
PEA HOMBRES QUINDIO 40-50      36603.539744234
PEA HOMBRES QUINDIO 60         27124.803442446
PEA MUJERES QUINDIO 18-24      7290.786458642
PEA MUJERES QUINDIO 24-30      17051.721509925
PEA MUJERE

# Mayor y menor tasa de desempleo por grupo de edad

### Risaralda

#### Hombres
- **Mayor tasa:** 24-30 años → **8.90%**
- **Menor tasa:** 60+ años → **3.41%**

#### Mujeres
- **Mayor tasa:** 18-24 años → **22.42%**
- **Menor tasa:** 30-40 años → **5.62%**

---

### Caldas

#### Hombres
- **Mayor tasa:** 18-24 años → **8.50%**
- **Menor tasa:** 24-30 años → **3.28%**

#### Mujeres
- **Mayor tasa:** 24-30 años → **10.85%**
- **Menor tasa:** 30-40 años → **5.03%**

---

### Tolima

#### Hombres
- **Mayor tasa:** 24-30 años → **14.66%**
- **Menor tasa:** 40-50 años → **4.75%**

#### Mujeres
- **Mayor tasa:** 18-24 años → **23.12%**
- **Menor tasa:** 60+ años → **13.83%**

---

### Quindío

#### Hombres
- **Mayor tasa:** 18-24 años → **16.27%**
- **Menor tasa:** 40-50 años → **4.57%**

#### Mujeres
- **Mayor tasa:** 24-30 años → **19.35%**
- **Menor tasa:** 60+ años → **3.34%**

#### Entre Risaralda, Caldas , Tolima y Quindío, cuál es el departamento con menor tasa de desempleo de mujeres entre los 18 y los 25 años ? 


In [103]:
DESEM_RISARALDA_18_25_M = df_des_edad_g[(df_des_edad_g['DPTO']==66 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 25) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN RISARALDA  18-25    {DESEM_RISARALDA_18_25_M}') 
PEA_RISARALDA_18_25_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==66) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 25) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES RISARALDA 18-25      {PEA_RISARALDA_18_25_M}')
DESEM_CALDAS_18_25_M = df_des_edad_g[(df_des_edad_g['DPTO']==17 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 25) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN CALDAS  18-25     {DESEM_CALDAS_18_25_M}') 
PEA_CALDAS_18_25_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==17) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 25) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES CALDAS 18-25      {PEA_CALDAS_18_25_M}')
DESEM_TOLIMA_18_25_M = df_des_edad_g[(df_des_edad_g['DPTO']==73 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 25) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN TOLIMA  18-25     {DESEM_TOLIMA_18_25_M}') 
PEA_TOLIMA_18_25_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==73) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 25) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES TOLIMA 18-25      {PEA_TOLIMA_18_25_M}')
DESEM_QUINDIO_18_25_M = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 25) & (df_des_edad_g['DSI']==1) & (df_des_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA MUJERES EN QUINDIO  18-25     {DESEM_QUINDIO_18_25_M}') 
PEA_QUINDIO_18_25_M = df_ft_edad_g[(df_ft_edad_g['DPTO']==63) & (df_ft_edad_g['FT']==1) & (df_ft_edad_g['P6040'] > 18) & (df_ft_edad_g['P6040'] < 25) & (df_ft_edad_g['P3271']==2)]['FEX_C18'].sum()
print(f'PEA MUJERES QUINDIO 18-25      {PEA_QUINDIO_18_25_M}')
TASA_D_R_18_25_M = (DESEM_RISARALDA_18_25_M/PEA_RISARALDA_18_25_M)*100
TASA_D_C_18_25_M = (DESEM_CALDAS_18_25_M/PEA_CALDAS_18_25_M)*100
TASA_D_T_18_25_M = (DESEM_TOLIMA_18_25_M/PEA_TOLIMA_18_25_M)*100
TASA_D_Q_18_25_M = (DESEM_QUINDIO_18_25_M/PEA_QUINDIO_18_25_M)*100
print(f'TASA DESEMPLEO MUJERES RISARALDA 18-25    {TASA_D_R_18_25_M}')
print(f'TASA DESEMPLEO MUJERES CALDAS 18-25       {TASA_D_C_18_25_M}')
print(f'TASA DESEMPLEO MUJERES TOLIMA 18-25       {TASA_D_T_18_25_M}')
print(f'TASA DESEMPLEO MUJERES QUINDIO 18-25      {TASA_D_Q_18_25_M}')


POBLACION DESMPLEADA MUJERES EN RISARALDA  18-25    5084.612956942
PEA MUJERES RISARALDA 18-25      25673.406726216002
POBLACION DESMPLEADA MUJERES EN CALDAS  18-25     1858.618688844
PEA MUJERES CALDAS 18-25      20518.647655742
POBLACION DESMPLEADA MUJERES EN TOLIMA  18-25     4964.2725597300005
PEA MUJERES TOLIMA 18-25      23864.75141406
POBLACION DESMPLEADA MUJERES EN QUINDIO  18-25     654.18919308
PEA MUJERES QUINDIO 18-25      9947.812534241999
TASA DESEMPLEO MUJERES RISARALDA 18-25    19.804979569579
TASA DESEMPLEO MUJERES CALDAS 18-25       9.058192918108219
TASA DESEMPLEO MUJERES TOLIMA 18-25       20.801693986241492
TASA DESEMPLEO MUJERES QUINDIO 18-25      6.5762115121105635


## Departamento con menor tasa de desempleo de mujeres (18–25 años)

Tasas de desempleo:

- **Risaralda:** 19.80%
- **Caldas:** 9.06%
- **Tolima:** 20.80%
- **Quindío:** 6.58%

### Resultado

El **departamento con menor tasa de desempleo de mujeres entre 18 y 25 años es Quindío (6.58%)**.

### Orden de menor a mayor

1. Quindío — 6.58%
2. Caldas — 9.06%
3. Risaralda — 19.80%
4. Tolima — 20.80%

##### ¿Cuántos hombres y mujeres hay en edad de trabajar en el quindio ? ¿Qué porcentaje de la población total representan ? 


In [90]:
PETQUINDIO = df_ft[(df_ft['DPTO']==63) & (df_ft['FT'])]['FEX_C18'].sum()
print(f'PET QUINDIO      {PETQUINDIO}')
POBLACION_TOTAL = df_cgsse['FEX_C18'].sum()
print(f'POBLACION TOTAL      {POBLACION_TOTAL}')
PORCENTAJE_PET_QUINDIO = (PETQUINDIO/POBLACION_TOTAL)*100
print(f'PORCENTAJE PET QUINDIO      {PORCENTAJE_PET_QUINDIO}')

PET QUINDIO      291511.75121534604
POBLACION TOTAL      52045001.000009045
PORCENTAJE PET QUINDIO      0.5601147960690699


##### POBLACION EN EDAD DE TRABJAR PARA EL QUINDIO ES DE 291.511
##### PORCENTAJE QUE REPRESENTAN PARA LA POBLACION TOTAL ES DEL 0.56%

##### ¿Qué porcentaje de la población del Tolima está en edad de trabajar ? 


In [92]:
POBLACION_DEL_TOLIMA = df_cgsse[df_cgsse['DPTO']==73]['FEX_C18'].sum()
print(f'POBLACION TOTAL TOLIMA      {POBLACION_DEL_TOLIMA}')
PET_TOLIMA = df_ft[(df_ft['DPTO']==73) & (df_ft['FT'])]['FEX_C18'].sum()
print(f'PET TOLIMA      {PET_TOLIMA}')
PORCENTAJE_PET_TOLIMA = (PET_TOLIMA/POBLACION_DEL_TOLIMA)*100
print(f'PORCENTAJE PET TOLIMA      {PORCENTAJE_PET_TOLIMA}')

POBLACION TOTAL TOLIMA      1411965.04428173
PET TOLIMA      662641.5050232499
PORCENTAJE PET TOLIMA      46.93044687663194


##### EL PORCENTAJE DE POBLACION EN EL TOLIMA QUE ESTA EN EDAD DE TRABAJAR ES DEL 46.93%

#### ¿ Qué porcentaje de la población de Risaralda es población económicamente inactiva ? 

In [94]:
POBLACION_DE_RISARALDA = df_cgsse[df_cgsse['DPTO']==66]['FEX_C18'].sum()
print(f'POBLACION TOTAL RISARALDA      {POBLACION_DE_RISARALDA}')
PEI_RISARALDA = df_ft[(df_ft['DPTO']==66) & (df_ft['FFT'])]['FEX_C18'].sum()
print(f'PEI RISARALDA      {PEI_RISARALDA}')
PORCENTAJE_PEI_RISARALDA = (PEI_RISARALDA/POBLACION_DE_RISARALDA)*100
print(f'PORCENTAJE PEI RISARALDA      {PORCENTAJE_PEI_RISARALDA}')

POBLACION TOTAL RISARALDA      1025240.9423573611
PEI RISARALDA      332743.42055744905
PORCENTAJE PEI RISARALDA      32.45514364578185


##### EL PORCENTAJE DE LA POBLACION ECONOMICAMENTE INACTIVA DE RISARALDA ES DEL 32.46%

#### ¿Cuántas personas hay en el Quindío en condición de desempleo entre los 18 y los 30 años ? 

In [96]:
DESEM_QUINDIO_18_30_H = df_des_edad_g[(df_des_edad_g['DPTO']==63 ) & (df_des_edad_g['P6040'] > 18) & (df_des_edad_g['P6040'] < 30) & (df_des_edad_g['DSI']==1)]['FEX_C18'].sum()
print(f'POBLACION DESMPLEADA EN QUINDIO  18-30     {DESEM_QUINDIO_18_30_H}') 

POBLACION DESMPLEADA EN QUINDIO  18-30     8683.081653568002


#### LA POBLACION DESEMPLEADA PARA EL QUINDIO ES DE 8.683 PERSONAS

#### ¿Cuántas personas hay ocupadas en el departamento del Tolima , cuyas edades están entre los 20 y los 25 años , qué porcentaje de la población total del Tolima representan ? 

In [101]:
OCU_TOL_20_25 = df_ocu_edad_g[(df_ocu_edad_g['DPTO']==73 ) & (df_ocu_edad_g['P6040'] > 20) & (df_ocu_edad_g['P6040'] < 25) & (df_ocu_edad_g['OCI']==1)]['FEX_C18'].sum()
print(f'POBLACION OCUPADA EN TOLIMA  20-25     {OCU_TOL_20_25}') 
PORCENTAJ_OCU_20_25_TOLIMA = (OCU_TOL_20_25/POBLACION_DEL_TOLIMA)*100 
print(f'PORCENTAJE OCUPADOS EN TOLIMA 20-25     {PORCENTAJ_OCU_20_25_TOLIMA}')

POBLACION OCUPADA EN TOLIMA  20-25     35049.89865228999
PORCENTAJE OCUPADOS EN TOLIMA 20-25     2.482348893426038


#### LA CANTIDAD DE OCUPADOS PARA EL TOLIMAE ES DE 35.049 
#### EL PORCENTAJE QUE REPRESENTAN LOS OCUPADOS EN EDADES DE 20 A 25 AÑOS ES DEL 2.48%